packages/application-components/src/experimental/components/product-picker-input/product-picker-input.tsx

```diff
@@ -155,7 +145,7 @@ const ProductPickerInput = (props: TProps): JSX.Element => {
   const prefetchSelectedProductQuery = useQuery(PrefetchProduct, {
     variables: { id: props.value },
     context: { target: GRAPHQL_TARGETS.COMMERCETOOLS_PLATFORM },
-    skip: !props.value,
+    skip: !props.value || !isUUID(props.value),
```

Comment:
now we skip the prefetch of product if the initial value was invalid.
this is preventive action to getting an `onInitialLoadError`